app/helpers/post_helper.rb

```diff
@@ -7,8 +7,10 @@ module PostHelper
   #
   # Params:
   # +post_path+::a path to an existing post on the website
```

Comment:
Again, add `ref` to the param list here with a short description.

Reply:
Fixed